pkgs/desktops/gnome-3/extensions/pop-shell/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub, glib, nodePackages }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-pop-shell";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev = version;
+    sha256 = "1ba1nrnk4cqgjx5mygqdkw74xlankrkiib9rw0vwkjcgv9bj024a";
+  };
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+  buildInputs = [ glib nodePackages.typescript ];
+
+  postInstall = ''
+    mv $out/usr/* $out
+    rmdir $out/usr
```

Comment:
if you add  ` makeFlags = [ "XDG_DATA_HOME=$(out)/share" ];` you don't need to move the files